fedot_ind/core/metrics/metrics_implementation.py

```diff
@@ -83,7 +83,7 @@ def metric(self) -> float:
         return mean_squared_error(
             y_true=self.target,
             y_pred=self.predicted_labels,
-            squared=False)
+            squared=False) ** 0.5
```

Comment:
в данном случае не то считается, по-моему
```suggestion
            squared=True) ** 0.5
```
или
```suggestion
            squared=False)
```
или можно использовать `root_mean_squared_error`, потому что параметр `squared` является deprecated в версиях старше `scikit-learn==1.4`